2024/02.md

```diff
@@ -35,7 +35,7 @@ Background:
 
 ## Agenda topic rules
 
-Deadline for advancement eligibility: [January 27th, 10:00 PST](https://www.timeanddate.com/countdown/generic?p0=1440&iso=20240127T18&msg=TC39%20Submission%20deadline)
+Deadline for advancement eligibility: [January 27th, 10:00 America/Los_Angeles](https://www.timeanddate.com/countdown/winter?iso=20240127T10&p0=137&msg=TC39+Submission+deadline&font=sanserif&csz=1)
```

Comment:
this one has always used the "PST" form instead of the long form (and the backslash would be needed to keep it)

Reply:
we use IANA everywhere else; just normalize it

Reply:
I don't disagree in principle, but I don't see it applying here.  not a hill I'm going to die on.   either way, prefer changing it to `PT` to leave the daylight saving bit out of it -- something that just adds risk of copypasta errors